freezeyt.py

```diff
@@ -2,12 +2,30 @@
 
 
 def url_to_filename(base, url):
+    """
+    Returns the filename to which the page is frozen
+
+    base is the path to the file
+    url is the link to the page
+    """
```

Comment:
```suggestion
    """Returns the filename path of frozen page.
    
    Parameters:
    base -- dirpath to the file
    url -- web app endpoint of page
    """
```

Reply:
jak psát docstringy jsem našel na https://www.python.org/dev/peps/pep-0257/

Reply:
Děkuji za návrh na změnu. PEP 257 jsem také četla a pochopila-li jsem ho správně, tak první řádek dokumentačního řetězce je možno psát na stejný řádek jako otevírací uvozovky, tak i na další řádek. Jelikož dokumentační řetězce, které byly v projektu před tím, než jsem se pustila do doplňování, používají variantu, kterou navrhuješ, takže já se přizpůsobím. Také jsem si všimla, že používám "špatný" tvar sloves v dokumentačních řetězcích, takže to budu ještě upravovat.
Chtěla bych se ještě doptat na pár věcí. Co je `filename path`? Co znamená `dirpath`? A zda tam opravdu mají být *paramentry* a ne *argumenty*?